### DISCUSSION TITLE
401 error updating table from Edge Function

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#26969] [Questions]
Hi,  I have a Edge function that is invoked by a 3rd Party System, the Edge function works but gets an 401 error when accessing a table or a DB-function.  JWT is enabled, and are sending a JWT bearer in the header of the request.  I have tested in Postman and get the same issue.

I have reduced the Edge function to just a test, and also ensured RLS is turned off on the table, but still get a 401 error.  I know the environment variables and set correctly as they are used by other edge functions that work fine.

![image](https://github.com/supabase/supabase/assets/122821259/3eefdc2e-cfaa-49f3-ae55-8d7246f3d402)

Any ideas anyone?, I have been stuck on this and cannot progress.

**Comment by GaryAustin1** [comment]
If you use your authorization header in Postman to do an insert to the Supabase server directly (not using your function) does it work?
Anymore detail on the 401 error message?  I believe it can be for RLS, Postgres grants and maybe even a bad jwt.
